Scoop/Controller/Home/NotificationsViewController.swift

```diff
@@ -66,6 +72,48 @@ class NotificationsViewController: UIViewController {
             make.bottom.equalTo(view.safeAreaLayoutGuide)
         }
     }
+
+    private func setupEmptyStateView() {
+        view.addSubview(emptyStateView)
+
+        emptyStateView.snp.makeConstraints { make in
+            make.top.bottom.leading.trailing.equalToSuperview()
+        }
+
+        let notifBellImageView = UIImageView()
+        notifBellImageView.image = UIImage.notifIcon
+        notifBellImageView.contentMode = .scaleAspectFit
+        notifBellImageView.layer.masksToBounds = true
+        emptyStateView.addSubview(notifBellImageView)
+
+        notifBellImageView.snp.makeConstraints { make in
+            make.centerX.equalToSuperview()
+            make.top.equalToSuperview().inset(340)
+            make.width.height.equalTo(50)
+        }
+
+        let noNewLabel = UILabel()
+        noNewLabel.text = "No new notifications"
+        noNewLabel.font = UIFont.bodySemibold
```

Comment:
Same as above with the `.scooped`. @rs929 can we refactor the codebase to use this for all constants?